### ISSUE TITLE
BOY changed how it resolves scripts from *.opi files in linked containers

### ISSUE THREAD

**Issue by kasemir** [ControlSystemStudio/cs-studio#998]
Similar to #977:

Assume you have `/path/to/shared/display_with_script.opi` that loads `some_script.py` from that same directory. Works fine.

Now you include that in a linking container from within `/other/path/main.opi`.
==> `some_script.py` not found, because it's resolved as `/other/path/some_script.py` instead of `/path/to/shared/some_script.py`

**Comment by kasemir** [comment]
Found a fix, but mostly learned that the `LinkingContainer` is problematic.

In principle, it includes a different `*.opi` file, with its own widget tree.
But loading an `*.opi` file merges all linked trees into one big widget tree, as if the `LinkingContainer` was like a `GroupingContainer` within the same tree.

This has always prevented us from simply implementing "open in editor" on a runtime, because the merged runtime model is different from the editor model.
#912 tried to fix some connector issue, added more `*Linking*` classes and a bunch of `instanceof` checks. #977 fixed one follow-up issue with relative display links. This fixes an issue with script paths, unclear if that's new or was always there.

Bottom line is that the whole `LinkingContainer` handling could be redone.
